extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -625,8 +633,12 @@ export class AmpAnalytics extends AMP.BaseElement {
    * @private
    */
   handleEvent_(trigger, event) {
-    if (trigger?.session?.['persistEvent'] && this.sessionManager_) {
-      this.sessionManager_.updateEventTimestamp(this.type_);
+    if (this.sessionManager_) {
+      this.sessionManager_?.updateEvent(
+        this.type_,
+        !!trigger.session?.['persistEvent'],
+        !!trigger.session?.['persistEngaged']
+      );
     }
```

Comment:
The conditional is redundant.
```suggestion
    this.sessionManager_?.updateEvent(
      this.type_,
      !!trigger.session?.['persistEvent'],
      !!trigger.session?.['persistEngaged']
    );
```